phi/document/reader/base.py

```diff
@@ -10,71 +12,11 @@ class Reader(BaseModel):
     chunk_size: int = 3000
     separators: List[str] = ["\n", "\n\n", "\r", "\r\n", "\n\r", "\t", " ", "  "]
 
+    def __init__(self, chunking_strategy: Optional[ChunkingStrategy] = None):
```

Comment:
@manthanguptaa we can't initialize this like this sadly, we need to send chunking strategy as a classvar above